Homework/src/by/tms/Lesson_6/Time.java

```diff
@@ -0,0 +1,73 @@
+package by.tms.Lesson_6;
+
+import java.util.ArrayList;
+
+/*
+Создать класс и объекты описывающие промежуток времени. Сам промежуток в классе должен задаваться тремя
+свойствами: секундами, минутами, часами. Сделать методы для получения полного количества секунд в объекте,
+сравнения двух объектов (метод должен работать аналогично compareTo в строках). Создать два конструктора:
+получающий общее количество секунд, и часы, минуты и секунды по отдельности. Сделать метод для вывода данных.
+ */
+
+public class Time {
+
+    private int seconds;
+    private int minutes;
+    private int hours;
+
+    public Time(int seconds) {
+        this.hours = seconds / 3600;
+        seconds %= 3600;
+        this.minutes = seconds / 60;
+        seconds %= 60;
+        this.seconds = seconds;
+    }
+
+    public Time(int hours, int minutes, int seconds) {
+        this.hours = hours;
```

Comment:
Convert please it in a proper way. 
Now it is possible to store Time object with 134 hours, 324324 minutes and 954388 seconds. But there should be not more than 60 minutes and 60 seconds in a result )